docs/nidmm/status.inc

```diff
@@ -7,7 +7,7 @@ NI-DMM Python API Status
 +===============================+======================+
 | Driver Version Tested Against | 19.1.0               |
 +-------------------------------+----------------------+
-| Driver API Version            | 19.6.0d0             |
+| Driver API Version            | 19.6.0d4             |
```

Comment:
You should integrate latest master and rebuild, since these are no longer added to the status.inc files.